DrinkingPub/Table.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vsite.Oom.DrinkingPub
+{
+    public class Table
+    {
+        private readonly int number;
+        private readonly string waiter;
+        private Order currentOrder;
+
+        public Table(int number, string waiter)
+        {
+            this.number = number;
+            this.waiter = waiter;
+        }
+
+        public void TakeOrder(Order order)
+        {
+            currentOrder = order;
```

Comment:
Vaš lokal će loše poslovati ako ćete pamtiti samo zadnju narudžbu za pojedini stol :-) Trebali biste pamtiti sve naružbe i čuvati ih u listi.